flask/src/test/resources/testProject/build.gradle

```diff
@@ -0,0 +1,55 @@
+plugins {
+    id 'net.corda.plugins.flask'
+}
+
+import net.corda.gradle.flask.FlaskJarTask
+import net.corda.gradle.flask.JavaAgent
+
+evaluationDependsOn('testAgent')
+
+configurations {
+    agent {
+        transitive = false
+    }
+}
+
+dependencies {
+    implementation project('testAgent')
+}
+
+File agentFile = project('testAgent').tasks.named('jar').get().outputs.files.singleFile
```

Comment:
You don't need `agentFile` any more, nor either of those two `import` statements.